BUILD.md

```diff
@@ -1,5 +1,5 @@
 # Important notice
-***In order to uniformize and simplify the build system we had to make choices. TC is currently only officially supported on Ubuntu 16.04 with gcc 5.4.0, cuda 9.0 and cudnn 7.***
```

Comment:
Why removing cuda/cudnn versions?  Pytorch will result in dynamic linking errors with other versions of cuda, and caffe2 (if built) will complain about cudnn (and you need _7.1_ specifically).

Reply:
The document now recommends installing a version of pytorch that corresponds to the system version of cuda, so it makes sense to remove this version here.